services/QuillLMS/config/locales/en.yml

```diff
@@ -34,8 +34,31 @@ en:
   clever:
     account_conflict: "This Clever account is already associated with another Quill account. Contact support@quill.org for further assistance."
   actioncontroller:
-    errors:
-      invalid_authenticity_token: 'Whoops, something went wrong. Please refresh your browser and try your request again.'
+  errors:
+    invalid_authenticity_token: 'Whoops, something went wrong. Please refresh your browser and try your request again.'
```

Comment:
I think this change (moving `errors` back a tab) is going to break anywhere that we're calling `actioncontroller.errors.invalid_authenticity_token`.